terraform/modules/cluster/gke-beta/scripts/kubernetes-setup.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+kubernetes-setup::install_drivers () {
+    kubectl get nodes --selector=cloud.google.com/gke-nodepool=${NODE_POOL_NAME} --no-headers \
+    | awk '{print $1}' \
```

Comment:
instead of separate `awk` step you can add `-o=name` to the previous `kubectl` command.

Reply:
actually there is difference between these 2
```
soumyapani@cloudshell:~ (supercomputer-testing)$ kubectl get nodes --selector=cloud.google.com/gke-nodepool=a3-multinic-ww-gke-nodepool-0 --no-headers -o=name
node/gke-a3-multinic-ww-g-a3-multinic-ww-g-3a81d541-jdxt
soumyapani@cloudshell:~ (supercomputer-testing)$ kubectl get nodes --selector=cloud.google.com/gke-nodepool=a3-multinic-ww-gke-nodepool-0 --no-headers | awk '{print $1}'
gke-a3-multinic-ww-g-a3-multinic-ww-g-3a81d541-jdxt
```